english/coffee/drink-water/drink.md

```diff
@@ -3,4 +3,8 @@ but you have to keep drinking,
 now your stomach swells to the size of a hot air balloon. 
 You start feeling an urge to release some pressure...
 
-Do you find the nearest [bathroom](bathroom/bathroom.md)?
+Do you want to:
+ 
+[Find the nearest bathroom](bathroom/bathroom.md)?
+
+[Go for a swim](Swim/swim.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.